app-center-webapps/src/main/webapp/skin/less/app-center.less

```diff
@@ -836,74 +840,113 @@
   max-height: 58px;
 }
 
-.appLauncherDrawerTitle {
-  font-size: 18px;
-  font-weight: bold;
-  color: @darkGrey;
-}
+.appCenterDrawer {
+  .title {
+    max-height: 57px;
+    .appLauncherDrawerTitle {
+      font-size: 18px;
+      font-weight: bold;
+      color: @darkGrey;
+    }
+  }
 
-.appLauncherList {
-  display: flex;
-  flex-wrap: wrap;
-  width: 100%;
-  padding: 20px 0;
-}
+  .appHeaderBorder {
+    border-color: #f0f4fe!important;
+  }
 
-.appLauncherItem {
-  flex: 0 1 33%;
-  a {
-    display: block;
-    padding: 5px;
+  .content {
+    max-height: 84vh;
+    overflow-y: auto;
+
+    .appLauncherList {
+      display: flex;
+      flex-wrap: wrap;
+      width: 100%;
+      padding: 20px 0;
+      overflow-y: auto;
+    }
+
+    .appLauncherItem {
+      flex: 0 1 33%;
+      a {
+        display: block;
+        padding: 5px;
 
-    &:hover .appLauncherImage {
-      box-shadow: -3px 3px 6px 0px #ececec;
-      transition: all .3s;
+        &:hover .appLauncherImage {
+          box-shadow: -3px 3px 6px 0px #ececec;
+          transition: all .3s;
+        }
+      }
+    }
+
+    .appLauncherImage {
+      width: 65px;
+      height: 65px;
+      display: block;
+      margin: 8px auto;
+      border: 1px solid #f0f4fe!important;
+      border-radius: 5px;
+      padding: 10px;
+    }
+
+    .appLauncherTitle {
+      display: block;
+      text-align: center;
+      color: #555759;
+      font-size: 16px;
     }
   }
-}
 
-.appLauncherImage {
-  width: 65px;
-  height: 65px;
-  display: block;
-  margin: 8px auto;
-  border: 1px solid #f0f4fe!important;
-  border-radius: 5px;
-  padding: 10px;
-}
+  .drawerActions {
+    padding-top: 15px;
+    padding-bottom: 15px;
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    width: 100%;
+    border-top: 1px solid #f0f4fe!important;
+    background: white;
 
-.appLauncherTitle {
-  display: block;
-  text-align: center;
-  color: #555759;
-  font-size: 16px;
+    .seeAllApplicationsBtn {
+      margin-bottom: 13px!important;
+      margin-top: 13px!important;
+      border: 1px solid #1976d27a!important;
+      padding: 8px 15px;
+      border-radius: 4px!important;
+      cursor: pointer!important;
```

Comment:
Are you sure this is mandatory ? On acceptance, if I remove this, the radius is still ok

Reply:
Fixed